artemis-hawtio/artemis-console/src/main/webapp/app/core/html/help.html

```diff
@@ -27,7 +27,7 @@
 
     #help-content
     {
-        position:absolute; left: 0; right: 0; bottom: 0; top: 90px;
```

Comment:
Just FYI, in my original work the help is our artemis documents as thats hosted in the web context anyhow, To see/view this you will need to do a full release build to make git book generate, this iframe loads into it the documents, avoiding us the need to have a separate help.